DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -631,8 +639,22 @@ function AnyEntryMapFixes()
             RemoveFears(hs);
             hs.MinHealth = 0;
         }
+
         Player().StartDataLinkTransmission("DL_FrontGate");
+
+        prevMapsDone = dxr.flagbase.GetBool('Heliosborn') && //Finished Vandenberg, mission 12
+            dxr.flagbase.GetBool('schematic_downloaded'); //Finished Ocean Lab, mission 14,
+        prevMapsDone = prevMapsDone || !#defined(injections) || dxr.flags.settings.goals<=0;
+        if(prevMapsDone && !dxr.flagbase.GetBool('MS_HowardStrongUnhidden')) {
+            foreach AllActors(class'#var(prefix)HowardStrong', hs) {
+                hs.EnterWorld();
```

Comment:
on line 348 we make howard LeaveWorld, you could move these conditions there so he only leaves world if the conditions aren't met